packages/next-swc/crates/next-core/src/app_source.rs

```diff
@@ -709,6 +709,8 @@ import {}, {{ chunks as {} }} from "COMPONENT_{}";
         let mut result =
             RopeBuilder::from("import { IPC } from \"@vercel/turbopack-node/ipc/index\";\n");
 
+        writeln!(result, "import \"../polyfill/app-polyfills.ts\";\n")?;
```

Comment:
Generate this relative path with a `get_relative_path` between `next_js_file_path("entry/app-renderer.tsx")` and `next_js_file_path("polyfill/app-polyfills.ts")` instead?